ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -176,27 +176,58 @@ Item {
                         spacing: 2
 
                         StatusBaseText {
+                            id: communityNameText
                             Layout.fillWidth: true
-
                             text: d.invitedCommunity.name
                             font.weight: Font.Bold
+                            maximumLineCount: 1
                             wrapMode: Text.WrapAtWordBoundaryOrAnywhere
```

Comment:
The `wrapMode` is no-op here if you set the `maximumLineCount` to 1 and also `elide`

Reply:
Fixed